include/sensors.h

```diff
@@ -94,6 +98,28 @@ class Sensors
     return true;
   }
 
+  static const float BARO_MAX_CHANGE_RATE;
+  static const float BARO_SAMPLE_RATE;
+  static const float DIFF_MAX_CHANGE_RATE;
+  static const float DIFF_SAMPLE_RATE;
+  static const float SONAR_MAX_CHANGE_RATE;
+  static const float SONAR_SAMPLE_RATE;
+  static const int SENSOR_CAL_DELAY_CYCLES;
+  static const int SENSOR_CAL_CYCLES;
+  static const float BARO_MAX_CALIBRATION_VARIANCE;
+  static const float DIFF_PRESSURE_MAX_CALIBRATION_VARIANCE;
```

Comment:
I did it for clarity. We could make at least the max variances a parameter for someone trying to get an especially bad sensor to work, but I don't think its necessary.